pkg/ecdsa/tss/member.go

```diff
@@ -3,10 +3,13 @@ package tss
 import (
 	"bytes"
 	"encoding/hex"
-	"github.com/keep-network/keep-core/pkg/operator"
 	"math/big"
+
+	"github.com/keep-network/keep-core/pkg/operator"
 )
 
+// FIXME How does this differ from an operator id? Why is it a distinct concept?
+// Improve name to reflect this.
```

Comment:
`MemberID` must be convertible to the network ID (`net.TransportIdentifier`). We need to know network IDs to open unicast channels between members. To obtain a network ID we need the operator public key and this is why `MemberID` looks as follows. Regarding the new structure, we should probably rename it to `MemberPublicKey` or `KeepMemberPublicKey` to not confuse it with `*ID` types. Alternatively, we could drop it and operate on bare `operator.PublicKey` but this is probably not for this PR as the refactoring will be rather massive.

Reply:
> we should probably rename it to MemberPublicKey or KeepMemberPublicKey

This is a great idea. If it's a public key, let's be clear that it's a public key. If it ceases to be a public key later on, we can look at why and adjust accordingly.